index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MANV Simulations-App</title>
```

Comment:
add the line: 
```html
<link rel="icon" type="image/x-icon" href="/webapp/favicon.png">
```
here to add the favicon from the flutter web app

Reply:
Done